code/src/containers/language-select/LanguageSelect.tsx

```diff
@@ -0,0 +1,49 @@
+import AppMenuItem from "@/components/app-menu-item/AppMenuItem";
+import AppSelect from "@/components/app-select/AppSelect";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import locales from "@/constants/locales";
+import { useLocaleContext } from "@/context/I18nProvider";
+
+import "@/containers/language-select/LanguageSelect.scss";
+
+const LanguageSelect = () => {
+  const { locale, setLocale } = useLocaleContext();
+
+  const localesItems = locales.map(({ translationKey, key, icon }) => {
+    const handleLocaleChange = () => {
+      setLocale(key);
+    };
+    return (
+      <AppMenuItem
+        key={key}
+        value={key}
+        onClick={handleLocaleChange}
+        className="language-select__item"
+      >
+        <svg className="language-select__item-icon">
+          <use href={icon} />
+        </svg>
+        <AppTypography translationKey={translationKey} />
+      </AppMenuItem>
+    );
+  });
+
+  return (
+    <AppSelect
+      className="language-select__container"
+      labelId="language-select"
+      IconComponent={() => null}
```

Comment:
Maybe we should keep icon here? I think for this select it would look better with caret icon for example

Reply:
Returned icon
![image](https://github.com/user-attachments/assets/53776f2f-1716-4f0c-9398-64bf6eaeab42)